.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+end_of_line = lf
```

Comment:
I'm not sure if it is a good idea to having this here... I'm not very familiar if Windows will complain

Reply:
I don't think you need this. Git will auto-convert for you and you may end up screwing up the file for the user. I don't think the Vim codebase is doing anything specific regarding newlines.